front/src/api/books.tsx

```diff
@@ -8,4 +8,9 @@ export const books = {
 
     return response;
   },
+  sendOrder: (data: object) => {
+    axios.post(`${config.urlSendOrder}`, data).catch(function (error) {
```

Comment:
jezeli cos sie tu wywali, to wyjątek i tak poleci wyzej, nie trzeba go tu lapac i rzucac jeszcze raz. Teoretycznie api tu powinno zwrocic dodany obiekt, albo liste z dodanym obiektem - to zalezy od tego jak jest zaprojektowane na backendzie. Zobacz co jest zwracane i to przeslij

Reply:
nie wiem czemu to tu wgl bylo chyba dlatego ze ucze sie axiosa dopiero i kopiowalem z neta i przeoczylem te lapanie errora. Zwracany jest obiekt ktory wysylam w formularzu wiec  usuwam to;
Done.